README.md

```diff
@@ -1,7 +1,26 @@
-# Getting Started with Create React App
+# A simple Calculator made with React
+
+The goal of the exercise is to create a working calculator with React.
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+![Screenshot](https://user-images.githubusercontent.com/31847346/148350518-a90c296d-972d-462f-aefa-6535dc70c9e5.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- React
+- JavaScript
+- Gitflow
+
+## Getting Started
+To get a local copy up and running, follow the steps in Setup below.
+
+### Prerequisites
+Basic knowledge of HTML, CSS, and JavaScript.
+
 ## Available Scripts
 
```

Comment:
- you did great adding `npm start` however As said by the previous reviewer please add `npm install` in your scripts to install `npm`